configure.ac

```diff
@@ -4221,6 +4221,7 @@ dnl bpo-45774/GH-29507: The CPP check in AC_CHECK_HEADER can fail on FreeBSD,
 dnl hence CPPFLAGS instead of CFLAGS.
   CPPFLAGS="$CPPFLAGS $LIBSQLITE3_CFLAGS"
   LDFLAGS="$LIBSQLITE3_LIBS $LDFLAGS"
+  LIBS="$LIBSQLITE3_LIBS $LIBS"
```

Comment:
We can replace `LDFLAGS` with `LIBS`. [I made a suggestion against your fork](https://github.com/zanieb/cpython/pull/1), and also incorporated adding `--static` to the pkg-config invokation if `--disable-shared` is given.

Reply:
Thanks! Do you mind if we use a separate pull request for discussion of

> also incorporated adding --static to the pkg-config invokation if --disable-shared is given.

It's a bit separate and I'll want to validate that separately.

Reply:
Notably, in `python-build-standalone`, we use a static sqlite even when building a shared libpython (https://github.com/astral-sh/python-build-standalone/blob/4a36797763f5cef939c8caeeed704bc08bab1530/cpython-unix/build-cpython.sh#L370)